README.md

```diff
@@ -10,7 +10,7 @@ This demonstrates how a simple json data source can be used in Grafana to read t
 
 ### Dependencies
 
-For native image support, graalvm 20.3.0 (Java 11 version) is needed with path to it's directory set via environment variable `GRAALVM_HOME`. This can be downloaded from:
+For native image support, graalvm 21.3.0 (Java 17 version) is needed with path to it's directory set via environment variable `GRAALVM_HOME`. This can be downloaded from:
```

Comment:
@andrewazores I have been wondering if there is a specific reason why we use `21.3.x` instead of new versions (maybe `v22.x`) of graalvm (same for native image)? I remember @ebaron suggests it in #111 but I am not sure to understand?

Reply:
I believe Quarkus 2.8.0 is where Graal 22+ is used. We're on the older Quarkus 2.7.0 so Graal 21.3 seems to be the correct dependency version.

Reply:
We generally try to use the same version upstream that we need to use in our product builds. This reduces the likelihood of bugs going undetected and makes our product builds easier to do. For Quarkus 2.7, this document explains that we should use version 21.3: https://access.redhat.com/articles/4966181#RHBQ_2_7_x

Interestingly the Java 17 version of the builder image is listed as tech-preview support (rather than GA/full support). This should be fine for us though, since our product builds are currently also tech-preview.

Reply:
ahh now that makes sense! Thanks a lot!